components/slack/actions/add_star/add_star.js

```diff
@@ -0,0 +1,42 @@
+const slack = require("../../slack.app.js");
+const { WebClient } = require("@slack/web-api");
+
+module.exports = {
+  key: "slack-add-star",
+  name: "Add Star",
+  description: "Add a star to an item on behalf of the authenticated user",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    slack,
```

Comment:
Just to confirm, these are all optional because I can add a star to conversations _or_ files, correct? How does the timestamp prop work in this context?

Reply:
Yes you are correct and the timestamp is used to identify the message within the conversation you'd like to star. I added descriptions for clarification.